.github/workflows/validate-pr_job.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Reusable workflow to validate a pull request
+
+on: workflow_call
+
+permissions:
+  contents: read
+  statuses: write
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Load super-linter configuration
+        run: cat .github/super-linter.env >> "$GITHUB_ENV"
```

Comment:
Does this make it possible to use different linting rules in different repos?

Reply:
It should. I did some testing [here](https://github.com/commerce-docs/commerce-xd-kits/actions/runs/13531479950). But it makes sense to check in this org as well, with a repo that uses a slightly different config.

Reply:
[Here](https://github.com/AdobeDocs/commerce-webapi/actions/runs/13532743628/job/37818426838?pr=404#step:4:2) the action loads the local config.
And [here](https://github.com/AdobeDocs/commerce-webapi/actions/runs/13532743628/job/37818426838?pr=404#step:5:169) it runs the NATURAL_LANGUAGE linter, which is not enabled in the commerce-php repo.